lib/pom.xml

```diff
@@ -149,6 +149,11 @@ Copyright(c) 2020 Futurewei Cloud
             <version>[30.0-jre,)</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-kubernetes</artifactId>
+            <version>2.12.0</version>
```

Comment:
Should we upgrade our ignite-core dependency to 2.12 as well? The master stays on 2.10.

``
        <dependency>
            <groupId>org.apache.ignite</groupId>
            <artifactId>ignite-core</artifactId>
            <version>2.10.0</version>
        </dependency>
``

Reply:
Good suggestions. will try it.

Reply:
In Medina, it used ignite lib8. it seems like 2.10 not works on kubernetes. @cj-chung